docs/sdks/client-side-identity.md

```diff
@@ -117,25 +125,25 @@ Here's what you need to know about the token auto-refresh:
+	- When identity has become invalid&#8212;for example, because the user has opted out.<br/>NOTE: The callback is *not* invoked when identity is temporarily unavailable and the auto-refresh keeps failing. In this case, the SDK continues using the existing advertising token as long as it hasn't expired.
 - A [disconnect()](#disconnect-void) call cancels the active timer. 
 
 ### Callback Function
 
 You can register functions to receive events from the UID2 SDK using the [Array Push Pattern](#array-push-pattern). There are a number of events currently available:
-- `SdkLoaded` will be raised once the SDK has been parsed and the global `__uid2` object has been constructed. This is useful for calling `init()`, especially if your script loading order is not guaranteed (e.g. you are using `async` or `defer` script loading).
-- `InitCompleted` will be raised when `init()` has finished and the SDK is ready for use. If an identity was provided in the `init` call, or the SDK was able to load a previously-provided identity, the identity will be included in the payload.
-- `IdentityUpdated` will be raised whenever there is a new identity available, or the existing identity is no longer available.
+- `SdkLoaded` is raised after the SDK has been parsed and the global `__uid2` object has been constructed. This is useful for calling `init()`, especially if your script loading order is not guaranteed (for example, if you are using `async` or `defer` script loading).
+- `InitCompleted` is raised when `init()` has finished and the SDK is ready for use. If an identity was provided in the `init` call, or the SDK was able to load a previously-provided identity, the identity is included in the payload.
+- `IdentityUpdated` is raised whenever there is a new identity available, or the existing identity is no longer available.
 
 :::tip
 You can provide as many callback functions as you want, and register them from anywhere. This allows you to split your code up in a way that makes sense for your site.
 :::
 
 #### Callback Function Signature
 
-Your callback function should accept two parameters: an event type and a payload. The type of the payload will depend on the event type.
+Your callback function should accept two parameters: an event type and a payload. The type of the payload depends on the event type.
 
-This is an example callback which handles the `SdkLoaded` event to call init, and the `InitCompleted` event to provide an identity if one isn't available once `init` has completed.
+The following example callback handles the `SdkLoaded` event to call `init` and then, if an identity isn't available once `init` has completed, calls the `InitCompleted` event to provide an identity.
```

Comment:
This re-wording isn't quite right, can we use:

> The following example callback handles the `SdkLoaded` event to call `init` and then, if an identity isn't available once `init` has completed, uses the `InitCompleted` event to provide an identity.

Reply:
Done. Thx!